test/e2e/installplan_e2e_test.go

```diff
@@ -2708,13 +2708,96 @@ var _ = Describe("Install Plan", func() {
 
 		installPlanName := subscription.Status.InstallPlanRef.Name
 
-		fetchedInstallPlan, err := fetchInstallPlanWithNamespace(GinkgoT(), crc, installPlanName, ns.GetName(), buildInstallPlanPhaseCheckFunc(operatorsv1alpha1.InstallPlanPhaseInstalling))
+		fetchedInstallPlan, err := fetchInstallPlanWithNamespace(GinkgoT(), crc, installPlanName, ns.GetName(), buildInstallPlanPhaseCheckFunc(operatorsv1alpha1.InstallPlanPhaseFailed))
 		require.NoError(GinkgoT(), err)
-		log(fmt.Sprintf("Install plan %s fetched with status %s", fetchedInstallPlan.GetName(), fetchedInstallPlan.Status.Phase))
+		log(fmt.Sprintf("Install plan %s fetched with phase %s", fetchedInstallPlan.GetName(), fetchedInstallPlan.Status.Phase))
+		log(fmt.Sprintf("Install plan %s fetched with conditions %+v", fetchedInstallPlan.GetName(), fetchedInstallPlan.Status.Conditions))
+	})
+
+	It("should fail an InstallPlan when multiple operatorgroups are present", func() {
+
+		log := func(s string) {
+			GinkgoT().Logf("%s: %s", time.Now().Format("15:04:05.9999"), s)
+		}
```

Comment:
This should probably use https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/ctx/ctx.go#L49 so that we can tweak test logging in one place.

Reply:
This coming over from an existing test but I can update these in my tests.
I'll do a follow up after this PR to make sure we're consistently using the same test logging util in the entire e2e suite.